app/controllers/tips_controller.rb

```diff
@@ -1,12 +1,8 @@
 class TipsController < ApplicationController
 	def index
-		@trip = Trip.find(params[:trip_id])
-		@tips = @trip.tips
-
-		respond_to do |format|
-			format.html { render }
-			format.json { render json: @tips }
-		end
+    @user = User.find_by(id: params[:user_id])
+    @trip = @user.trips.find_by(id: params[:trip_id])
```

Comment:
hey @ap2539 , can you tell me a little more about why you changed this so that it looks for the user first, instead of just taking the trip_id param? because what if the user is looking at a friend's trip's tips? wouldn't the user_id params show the current user, not the friend's id?